src/pages/Confirm/Confirm.styled.jsx

```diff
@@ -1,11 +1,22 @@
 import styled from '@emotion/styled';
 
+export const RatingTitle = styled.span`
+  font-weight: bold;
+  font-size: 24px;
+  color: #005D9D;
+  margin-bottom: 10px;
+  margin-top: 25px;
+  line-height: 23px;
+  font-weight: bold;
+  label: rating-title;
+`
+
 export const RatingImg = styled.img`
-  max-width: 100px;
+  max-width: 60%;
```

Comment:
Sorry, @Katsu08 can you check if this value is OK, please?

Reply:
That never apply since the container has a `min-width: 300px`

![imagen](https://user-images.githubusercontent.com/66505715/157103628-fccfefaf-cc86-4a21-a228-fb08b4ce550d.png)

Then container is fixes to 300px 
and image max-width is fixes to 180px (because the image is 60% of container)

![imagen](https://user-images.githubusercontent.com/66505715/157103866-c4e6f602-4964-4adf-ad6d-bc0b07cdeeab.png)

You would use media queries to do the image more little on small screens

![imagen](https://user-images.githubusercontent.com/66505715/157104122-d43d74b2-edf2-4256-a751-919b6c13bf08.png)